pkgs/development/compilers/cudatoolkit/common.nix

```
@@ -135,6 +135,7 @@ stdenv.mkDerivation rec {
     rm -f $out/lib64/libOpenCL*
     ${lib.optionalString (lib.versionAtLeast version "10.1") ''
       mv $out/lib64 $out/lib
+      cp $out/extras/CUPTI/lib64/libcupti* $out/lib
```

```suggestion
      mv $out/extras/CUPTI/lib64/libcupti* $out/lib
```

Moving or copying usually doesn't really matter - the build directory is ephemeral. However, here it causes copies of the library to be created - while it could be symlinks:
```
lrwxrwxrwx 1 root root        16 Jan  1  1970 libcupti.so -> libcupti.so.10.2
lrwxrwxrwx 1 root root        19 Jan  1  1970 libcupti.so.10.2 -> libcupti.so.10.2.75
-r-xr-xr-x 1 root root   5785112 Jan  1  1970 libcupti.so.10.2.75
```

---

In this case I'm cp/mv'ing from $out to $out. And this leaves an orphan directory. Either way, commit updated.